app/models/form526_submission.rb

```diff
@@ -620,4 +662,8 @@ def submit_flashes
   def cleanup
     EVSS::DisabilityCompensationForm::SubmitForm526Cleanup.perform_async(id)
   end
+
+  def last_remediation
+    form526_submission_remediations&.order(created_at: :desc)&.last
```

Comment:
I think we want `:asc` if using `.last`? 🤔

Reply:
Good catch! Corrected in [ccd6694](https://github.com/department-of-veterans-affairs/vets-api/pull/17274/commits/ccd66940f6df82e0f004b5fdce9c60a6bda9be70).